package/src/components/DataTable/DataTable.md

```diff
@@ -0,0 +1,284 @@
+### Overview
+
+The DataTable provides a robust solution for displaying tabular data. It is built using [react-table](https://github.com/tannerlinsley/react-table) with the Material-UI [Table](https://material-ui.com/components/tables/) components.
+
+### Usage
+
+#### Types
+
+##### Standard selectable table data fetching and filtering
```

Comment:
This title is a little bit of a mouthful. Maybe we can just call this the `Standard table`. The paragraph below can explain that the `Standard table` has selectable rows, data-fetching and filtering.